config/management/genesis/README.md

```diff
@@ -0,0 +1,190 @@
+# Libra Config Manager
+
+The Libra Config Manager provides a tool for end-to-end management of the Libra
```

Comment:
nit: Maybe update this description to be more specific about genesis and remove "maintenance"?

Reply:
Ah, I straight copied it from the previous README